### ISSUE TITLE
On hiveos it keeps calling all my gpus dead

### ISSUE THREAD

**Issue by dahmsarmy** [todxx/teamredminer#467]
Running latest team red miner on hiveos and since the last update it’s shutting down and restarting after saying my gpus are dead.  Have tried downgrading but still throws errors but instead of all of them it only calls out a random gpu and then shuts down.

**Comment by darkgodbm** [comment]
See if this works the same for you: let the miner start, stop it as soon as it does. Change something in any of your cards (voltage, frequency, doesn't matter), apply, refresh. Restart your miner. Check if it'll run for a longer time before restarting again. Mine does for days, sometimes. What I can also confirm is that I get lower power consumption (about 10-12w less) in a fresh reboot, but higher if I pause/change/unpause. Really can't say what is causing this.

**Comment by dahmsarmy** [comment]
I’ll give that a shot.  Thank you

**Comment by darkgodbm** [comment]
So, how did it work for you? Did you try something else, too? Please let me know.

**Comment by dahmsarmy** [comment]
Sorry.  I ment to respond again.  I couldn’t get it to not throw a dead gpu error.  I switched to lolminer and it’s been stable 98% of the time so far.

**Comment by darkgodbm** [comment]
> Sorry. I ment to respond again. I couldn’t get it to not throw a dead gpu error. I switched to lolminer and it’s been stable 98% of the time so far.
> […](#)
> Sent from my iPhone
> On Nov 23, 2021, at 22:49, darkgodbm ***@***.***> wrote: ﻿ So, how did it work for you? Did you try something else, too? Please let me know. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android.

Yeah I did manage to try that one too but I can't get the MH/s I do with trm on my 5700's, not unless I draw a lot more power and that just won't do it for me.

**Comment by dahmsarmy** [comment]
I’m still running the xfx rx 580’s.  But have 4 5700’s on their way with 2 1300 watt evga super nova psu.

**Comment by dahmsarmy** [comment]
Wanted to give you an update.  The system has been non stop crashing the last 4 days.   Figured out that my CPU was crapping out.  I got a new one today and going on 3hrs with no issues at all so far! Knock on wood!!!
